src/flexbox/nasa.css

```diff
@@ -0,0 +1,60 @@
+.main-container {
+    
+}
+
+.nasa {
+    height: 100%;
+    width: 100%;
+    margin-bottom: 20px;
+    background-image: url(./space.jpg);
+    background-size: cover;
+    background-position: cover;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+
+}
+
+h2 {
+    color:white;
+    font-size: 150px;
+    font-family: fantasy;
+    letter-spacing: 10px;
+}
+
+#message {
+    height:50px;
+    margin-bottom: 10px;
+}
+
+#name {
+    margin-bottom: 10px;
+}
+
+.social {
+    background-color: rgb(146, 141, 141);
+    display: flex;
+    height: 35%;
+    width: 100%;
+    justify-content: space-evenly;
+}
+
+h3 {
+    font-size: 30px;
+    margin-top: 30px;
+    margin-bottom: 30px;
+    font-family:'Segoe UI', Tahoma, Geneva, Verdana, sans-serif
+}
+
+li {
+    list-style-type: none;
+    margin-bottom: 5px;
+}
+
+@media (max-width:800px) {
+    .social {
+        display:block;
```

Comment:
since you're using flex, stick with flex. remove this and add `flex-direction: column;`. Also set `height: 100%;`. when you do this, it should be in a column. but each `<ul>` would look better if the text were center. add flex to `ul` and center everything inside.